tools/tests/dropdown/test.js

```diff
@@ -183,6 +183,39 @@ describe('Dropdown Tests', () => {
             });
         });
 
+        it('Should scroll to the selected option when options list opens', async () => {
+            const dropdown = document.querySelector('gameface-dropdown');
+            const scrollableContainer = dropdown.querySelector('.guic-scrollable-container');
+            const selectedElPlaceholder = dropdown.querySelector('.guic-dropdown-selected-option');
+
+            const option = dropdown.querySelector('dropdown-option');
+
+            let selectedIndex = 3;
+
+            // Check if the selected option is disabled. This is a fallback in case the template is changed.
+            const selectedOption = () => {
+                const option = dropdown.allOptions[selectedIndex];
+
+                if (option.disabled) {
+                    selectedIndex++;
+                    return selectedOption();
+                }
+
+                return option;
+            };
+
+            dropdown.value = selectedOption().textContent;

```

Comment:
I think this is a good candidate for a util method. You can move this outside this test so it can be used by the other tests :D

Reply:
I have no idea what it can be used besides this :D I will skip adding it for now and we can revisit later.